### DISCUSSION TITLE
k33

### DISCUSSION THREAD

**Discussion by zikorayann** [Chia-Network/bladebit#100] [Q&A]
When will it be possible to create k-33 plots?

**Comment by zikorayann** [comment]
http://joxi.ru/ZrJZq0YibR4ZNr
![image](https://user-images.githubusercontent.com/26109769/133613300-d8a2353d-498a-4019-95c8-04523f8dc2c5.png)
I have a server with 2 TB of RAM and 2 x Intel (R) Xeon (R) Platinum 8259CL CPU @ 2.50GHz
I can help with testing and provide access to the server :)

**Comment by harold-b** [comment]
Nice setup!
K33 will not be supported, at least not in Bladebit's current form. Perhaps in the future as a different type of plotter because of it fundamentally changes the setup.
